mysite/__init__.py

```diff
@@ -0,0 +1 @@
+"""Initialization module."""
```

Comment:
Nope. The docstring in `__init__` should about the `mysite` package. Also, `Initialization module` doesn't actually mean anything useful.
Imagine you open this file and expect to see some hint on what's inside `mysite/` — will this sentence help you?

Reply:
Initializes the mysite directory as a standard python package? So it will be right?

Reply:
No.
1) You don't need to explain how Python works. It's not a Wikipedia article.
2) "Initializes" is a verb. It's a module. A declaration. NOT an action. It does not DO anything, it IS something.

Reply:
What should I write?

Reply:
Something like `Django website package.`

But the ultimate goal of this all going back-and-forth with choosing docstrings is to make you understand the ground rules and learn to make useful docstrings.

Reply:
Ok. Thanks.